tools/reactive-controllers/src/FocusGroup.ts

```diff
@@ -145,6 +155,24 @@ export class FocusGroupController<T extends HTMLElement>
         );
     }
 
+    changeDefaultItemFocus(): void {
```

Comment:
Please test that deleting something that is _NOT_ currently focused doesn't have negative effects on the focus here.

If would be great if we could do some work with the `disabled` attribute, but it is fine if we hold that change for a follow up PR.

Reply:
can you elaborate this? I didn't understand it

Reply:
This make sense we can add an attribute property in Mutation Observer config to check for updates which have disabled property and not do this on the method level. I can write a new ticket for this.

Reply:
Great!